### PR TITLE
e2e(react-nested-routers sample)

### PR DESCRIPTION
**Pull request by PeaceDate**
Screenshot 2022-12-13 at 18 05 15

### CODE REVIEW

File: cypress/config/cypress.config.ts [hunk 1]
```diff
@@ -14,13 +14,13 @@ export default defineConfig({
     specPattern: './**/e2e/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: './cypress/support/e2e.ts',
     fixturesFolder: './cypress/fixtures',
-    defaultCommandTimeout: 10000,
+    defaultCommandTimeout: 10_000,
```
Comment (edchai) [inline cypress/config/cypress.config.ts]: please revert. It's some strange idea from bot

File: react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        page2: string,
+        host: number
+    }
+) => {
+    describe(`Check ${property.appName}`, () => {
+        beforeEach(() => {
+            basePage.openLocalhost(property.host)
+        })
+    
+        it(`Check ${property.appName} build and running (Check elements on the page)`, () => {
```
Comment (edchai) [inline react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts]: please specify in name which element on page checked

File: react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+})
+
+describe('Check App 3', () => {
+    const navigation = Constants.elementsText.reactNestedRoutersNav
+
+    beforeEach(() => {
+        basePage.openLocalhost(8080)
+    })
+
+    it('Check App 3 build and running (Check elements on the page)', () => {
```
Comment (edchai) [inline react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts]: same comment for naming

**Review by edchai — CHANGES_REQUESTED** [review]

**Comment by ScriptedAlchemy** [comment]
needs rebase

**Comment by ScriptedAlchemy** [comment]
needs rebase

File: cypress/config/cypress.config.ts (hunk 1 above)
Comment (PeaceDate) [reply cypress/config/cypress.config.ts]: Fixed

File: react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts (hunk 2 above)
Comment (PeaceDate) [reply react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts]: Fixed

File: react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts (hunk 3 above)
Comment (PeaceDate) [reply react-nested-routers/e2e/checkReactNestedRoutersApps.cy.ts]: Fixed

**Review by edchai — APPROVED** [review]

**Pull request merged**